### PR TITLE
Adding more Skrell clothes

### PR DESCRIPTION
**Pull request by WhatsUpBrotendo**
Adds a casual wetsuit for Skrell.
Adds tailbands with SCS symbols on them for Skrell.
Adds nation capes for Skrell and Dionae.

Credits to Ryverstyx for the tailband and cape sprites.

image
image
image

While I did test everything, there is an issue with the wetsuit where parts of it will always be on top of accessories put onto the uniform. I think it's something to do with how the dual-colouring code works rather than my implementation of it. It's not game-breaking but if it's an issue that would mean removing it from the PR until fixed please let me know.

### CODE REVIEW

**Comment by WhatsUpBrotendo** [comment]
!review

**Review by KingOfThePing — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

File: code/skrell.dm [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+	if((slot in list(slot_head, slot_l_ear, slot_r_ear)) && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(istype(hair_styles_list[H.h_style], /datum/sprite_accessory/hair/skr_tentacle_m))
+			var/datum/sprite_accessory/hair/skr_tentacle_m/hair_datum = hair_styles_list[H.h_style]
+			if(hair_datum.scrunchy_style)
+				item_state = "scrunchy_[hair_datum.scrunchy_style]"
+	return ..()
+
+/obj/item/clothing/ears/skrell/workcap
+	name = "Worker's Cap"
```
Comment (listerla) [inline code/skrell.dm:212]:
```suggestion
	name = "worker's cap"
```

File: code/skrell.dm [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+	desc = "A simple clothing item used by Skrell to cover their headtails. It comes with a main sleeve for the middle headtail, with smaller sleeves for the outer headtails to help keep it in place."
+	icon = 'icons/obj/contained_items/skrell/jargon_caps.dmi'
+	item_state = "skrell_cap"
+	icon_state = "skrell_cap_item"
+
+/obj/item/clothing/ears/skrell/workcap/long
+	name = "Long Worker's Cap"
```
Comment (listerla) [inline code/skrell.dm:219]:
```suggestion
	name = "long worker's cap"
```

File: code/skrell.dm [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+	desc = "A simple clothing item used by Skrell to cover their headtails. It comes with a main sleeve for the middle headtail, with smaller sleeves for the outer headtails to help keep it in place. This one is longer to account for Skrell with longer headtails."
+	item_state = "skrell_cap_long"
+
+/obj/item/clothing/ears/skrell/tailband/
+	name = "Ox Tailband"
```
Comment (listerla) [inline code/skrell.dm:224]:
```suggestion
	name = "\improper Ox tailband"
```

File: code/skrell.dm [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+	desc = "A band meant to be worn on a Skrell's main headtail. This one has the ox symbol on it."
+	icon = 'icons/obj/contained_items/skrell/tailband.dmi'
+	item_state = "ox"
+	icon_state = "tailband"
+
+/obj/item/clothing/ears/skrell/tailband/ix
+	name = "Ix Tailband"
```
Comment (listerla) [inline code/skrell.dm:231]:
```suggestion
	name = "\improper Ix tailband"
```

File: code/skrell.dm [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+	desc = "A band meant to be worn on a Skrell's main headtail. This one has the ix symbol on it."
+	item_state = "ix"
+	icon_state = "tailband"
+
+/obj/item/clothing/ears/skrell/tailband/oqi
+	name = "Oqi Tailband"
```
Comment (listerla) [inline code/skrell.dm:237]:
```suggestion
	name = "\improper Oqi tailband"
```

File: code/skrell.dm [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+	desc = "A band meant to be worn on a Skrell's main headtail. This one has the oqi symbol on it."
+	item_state = "oqi"
+	icon_state = "tailband"
+
+/obj/item/clothing/ears/skrell/tailband/iqi
+	name = "Iqi Tailband"
```
Comment (listerla) [inline code/skrell.dm:243]:
```suggestion
	name = "\improper Iqi tailband"
```

**Comment by listerla** [comment]
Could also capitalize the national names for the capes and hit them with \improper as well.

**Comment by listerla** [comment]
You did /improper instead of \improper on the national capes - means they would have the "/improper" show on examine.

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 7]
```diff
@@ -379,3 +379,46 @@ datum/gear_tweak/social_credit/tweak_item(var/obj/item/clothing/accessory/badge/
 	dress["maelstrom formal wetshirt"] = /obj/item/clothing/accessory/dressshirt/skrell/maelstrom
 	dress["reef formal wetshirt"] = /obj/item/clothing/accessory/dressshirt/skrell/reef
 	gear_tweaks += new /datum/gear_tweak/path(dress)
+
+/datum/gear/uniform/skrell/wetsuit
+
+	display_name = "Casual Wetsuit"
```
Comment (Llywelwyn) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:385]:
```suggestion
	display_name = "casual wetsuit"
```

**Review by Llywelwyn — CHANGES_REQUESTED** [review]
looks good. i think some of the code is written to a wrong file by mistake, but what is there is good once copied to the right place. and just commented two pedantic changes to case - for whatever reason, our loadout is always lower case, even for proper nouns. it's only acronyms that are in caps

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 8]
```diff
@@ -379,3 +379,46 @@ datum/gear_tweak/social_credit/tweak_item(var/obj/item/clothing/accessory/badge/
 	dress["maelstrom formal wetshirt"] = /obj/item/clothing/accessory/dressshirt/skrell/maelstrom
 	dress["reef formal wetshirt"] = /obj/item/clothing/accessory/dressshirt/skrell/reef
 	gear_tweaks += new /datum/gear_tweak/path(dress)
+
+/datum/gear/uniform/skrell/wetsuit
+
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:384]:
```suggestion
```

File: code/modules/clothing/ears/skrell.dm [hunk 9]
```diff
@@ -219,3 +219,28 @@
 	name = "Long Worker's Cap"
 	desc = "A simple clothing item used by Skrell to cover their headtails. It comes with a main sleeve for the middle headtail, with smaller sleeves for the outer headtails to help keep it in place. This one is longer to account for Skrell with longer headtails."
 	item_state = "skrell_cap_long"
+
+/obj/item/clothing/ears/skrell/tailband
+	name = "Ox Tailband"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/ears/skrell.dm:224]:
```suggestion
	name = "ox tailband"
```

File: code/modules/clothing/ears/skrell.dm [hunk 10]
```diff
@@ -219,3 +219,28 @@
...
+	desc = "A band meant to be worn on a Skrell's main headtail. This one has the ox symbol on it."
+	icon = 'icons/obj/contained_items/skrell/tailband.dmi'
+	item_state = "ox"
+	icon_state = "tailband"
+
+/obj/item/clothing/ears/skrell/tailband/ix
+	name = "Ix Tailband"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/ears/skrell.dm:231]:
```suggestion
	name = "ix tailband"
```

File: code/modules/clothing/ears/skrell.dm [hunk 11]
```diff
@@ -219,3 +219,28 @@
...
+	desc = "A band meant to be worn on a Skrell's main headtail. This one has the ix symbol on it."
+	item_state = "ix"
+	icon_state = "tailband"
+
+/obj/item/clothing/ears/skrell/tailband/oqi
+	name = "Oqi Tailband"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/ears/skrell.dm:237]:
```suggestion
	name = "oqi tailband"
```

File: code/modules/clothing/ears/skrell.dm [hunk 12]
```diff
@@ -219,3 +219,28 @@
...
+	desc = "A band meant to be worn on a Skrell's main headtail. This one has the oqi symbol on it."
+	item_state = "oqi"
+	icon_state = "tailband"
+
+/obj/item/clothing/ears/skrell/tailband/iqi
+	name = "Iqi Tailband"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/ears/skrell.dm:243]:
```suggestion
	name = "iqi tailband"
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
You did not capitalize ox/iqi, so I assume that they are not proper names. Neither is "tailband".

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**